LICENSE

```diff
@@ -1,3 +1,98 @@
+Copyright (c) 2014, Vaibhav Singh (design) and Rosetta Type Foundry s.r.o. (post-production).
```

Comment:
Is this file supposed to be here? looks like there is another license file under `rally_assets/LICENSE`

Reply:
I believe that we want to keep one LICENSE file for the whole repo in the root directory that includes all licenses and the reason we have a separate in the rally_assets directory, as well as the shrine_images directory, is to make sure that it gets added whenever someone is downloading the package.

Reply:
ah gotcha, I originally thought this was a new file and didn't realize that it was amending the existing one. Cool